packages/transaction-controller/src/utils/gas.ts

```diff
@@ -172,7 +172,7 @@ async function requiresFixedGas({
   }
 
   if (!to) {
-    return true;
+    return false;
```

Comment:
Brilliant spot!

Can we simplify this into line 170 and change it to:

`if (isCustomnetwork || !to || data)`

That way we only do the `getCode` request when we absolutely have to?

Reply:
That is great I love it! Pushed the changes. :pray: